src/main/webapp/app/course/learning-paths/participate/learning-path-graph-sidebar.component.scss

```diff
@@ -0,0 +1,77 @@
+@import 'src/main/webapp/content/scss/artemis-variables';
+
+$draggable-width: 15px;
+$graph-min-width: 215px;
+
+.learning-path-sidebar {
+    .expanded-graph {
+        display: flex;
+        width: calc(#{$draggable-width} + #{$graph-min-width});
+        min-height: 500px;
```

Comment:
Are learning paths intended to be viewed on mobile devices?

Reply:
Participating in LPs on a mobile device is not planned. If it's deemed necessary further down the line I would address this in a separate PR.